src/js/mainContainer.js

```diff
@@ -331,14 +337,48 @@ class MainContainer extends React.Component {
         return returnAllData;
     }
     ectInsertSC() {
-        if (typeof window.ectWPInsertSC != "undefined") {
-            window.ectWPInsertSC();
-        }
+        // if (typeof ectInsertSC != "undefined") {
+        //     ectInsertSC();
+        // }
+        var params = {
+            'timerName': this.state.naMeP,
+            'userID': 1,
+            'fontSize': this.state.fontSize,
+            'fontSizeTxt': this.state.fontSizeTxt,
+            'color': this.state.pColor,
+            'colorTxt': this.state.pColorTxt,
+            'isBold': this.state.pIsBold,
+            'isBoldTxt': this.state.pIsBoldTxt,
+            'timezoneOffset': this.state.timezoneOffset,
+            'endHour': this.state.endHour,
+            'endMinute': this.state.endMinute,
+            'utcTz': this.state.utcTz,
+            'yearsTxt': this.state.yearsFormat,
+            'monthsTxt': this.state.monthsFormat,
+            'weeksTxt': this.state.weeksFormat,
+            'daysTxt': this.state.daysFormat,
+            'hoursTxt': this.state.hoursFormat,
+            'minutesTxt': this.state.minutesFormat,
+            'secondsTxt': this.state.secondsFormat,
+            'customEndedTxt': this.state.customTxtEndedTxt,
+            'layoutType': this.state.layoutType
+        };
+        console.log(params, "params");
+
+        axios.put('http://localhost/wordpress/wp-json/ect/v2/addTimer', params)
```

Comment:
replace this with axios.put(ectWPPath+'/wp-json/ect/v2/addTimer', params)
be sure to add to index.html the following:
 var ectWPPath = "http://localhost/wordpress";